lute/std/libs/path/posix/init.luau

```diff
@@ -151,10 +155,10 @@ function posix.normalize(path: pathlike): path
 		end
 	end
 
-	return {
+	return setmetatable({
 		parts = newParts,
 		absolute = path.absolute,
-	}
+	}, posix)
```

Comment:
should the metatable be a separate table, not `posix` itself? it feels like there's a conflation happening between the library of functions and the path object itself